packit/config/package_config.py

```diff
@@ -189,6 +189,23 @@ def get_all_files_to_sync(self):
 
         return SyncFilesConfig(files)
 
+    def get_copr_build_project_value(self) -> Optional[str]:
+        projects_list = [
+            job.metadata.get("project")
+            for job in self.jobs
+            if job.type == JobType.copr_build and job.metadata.get("project")
+        ]
+        if not projects_list:
+            return None
+
+        if len(set(projects_list)) > 1:
+            logger.warning(
+                f"You have defined multiple copr projects to build in, we are going "
+                f"to pick the first one: {projects_list[0]}, reorder the job definitions"
+                f" if this is not the one."
```

Comment:
```suggestion
                f" if this is not the one you want."
```

Reply:
I specifically omitted those two words b/c I assumed that's implied.

Reply:
Added

Reply:
> I specifically omitted those two words b/c I assumed that's implied.

Sorry. Do you mind being explicit?